padigitale-report-generator/src/main/resources/application.conf

```
@@ -0,0 +1,19 @@
+report-generator {
+  storage {
+    interfaces-container = ${INTERFACES_CONTAINER}
```

since we are refactoring, may I suggest this standardization?
```suggestion
    eservices-interfaces {
      container = ${INTERFACES_CONTAINER}
    }
```